lib/node_modules/@stdlib/math/base/special/fresnel/README.md

```diff
@@ -0,0 +1,76 @@
+# fresnel
+
+> Compute the [Fresnel integral][fresnel-integral].
+
+<section class="usage">
+
```

Comment:
TODO: add equation.

Reply:
✅

Reply:
Added the surrounding code. Should I programatically insert the SVG, or is that best left to a different step in the process?

Reply:
(Edit: I'd actually kinda like to insert this myself so I can verify the formatting/correctness.)

Reply:
The SVGs can be automatically generated and inserted via the command `make markdown-equations`. AFAIK, it's not possible right as of now to generate them only for a certain folder, so maybe best not to run it as part of this pull request. To check formatting/correctness, I have in the past used [this LaTeX equation editor](https://www.codecogs.com/latex/eqneditor.php).

Reply:
I think I did succeed running

```bash
$ make MARKDOWN_FILTER=.*/math/base/special/fresnel/.* markdown-*
```

type commands. Though it may definitely be best not to run it.

Reply:
@Planeshifter As @rreusser showed in his example, it is possible to generate equations for specific folders. This can be done using the `MARKDOWN_FILTER` and `MARKDOWN_PATTERN` environment variables. Alternatively, to bypass resolution of Markdown files, one can supply a list of Markdown files directly with the `MARKDOWN_FILES` environment variable. If providing a list of Markdown files, I would advise providing absolute filepaths.